### DISCUSSION TITLE
How to enable border in completion window

### DISCUSSION THREAD

**Discussion by FahimAnayet** [VonHeikemen/lsp-zero.nvim#102] [Q&A] [answered]
I have used both the following way, it's not working. Am I missing something?
```
local lsp = require('lsp-zero')
lsp.preset('recommended')

local cmp = require('cmp')
lsp.setup_nvim_cmp({
  completion = {
    border = 'rounded',
  }
})
lsp.setup()
```

```
cmp.setup({
window = {
completion = cmp.config.window.bordered(),
}})
```

**Comment by VonHeikemen** [comment, marked as answer]
This should work.

```lua
local lsp = require('lsp-zero')
lsp.preset('lsp-compe')

lsp.setup()

vim.opt.completeopt = {'menu', 'menuone', 'noselect'}

local cmp = require('cmp')
local cmp_config = lsp.defaults.cmp_config({
  window = {
    completion = cmp.config.window.bordered()
  }
})

cmp.setup(cmp_config)
```

**Reply by FahimAnayet** [reply]
This worked. Thank you.

**Reply by josepheiba** [reply]
<img width="225" alt="Screenshot 2024-01-31 at 6 55 48 AM" src="https://github.com/VonHeikemen/lsp-zero.nvim/assets/101673502/54030715-14da-46bf-a8bc-bdd79557da0f">
Hi, please let me know how to remove the border from the Documentation window
window = {
      documentation = cmp.config.window. ...
  },

**Reply by VonHeikemen** [reply]
@josepheiba if you are using `v3.x` of lsp-zero, you need to set the global variable `lsp_zero_ui_float_border` to `0` before the plugin is loaded.

```lua
vim.g.lsp_zero_ui_float_border = 0
```

**Reply by josepheiba** [reply]
Thank you very much

**Reply by josepheiba** [reply]
sorry about my deleted comment, I have caused the new issue, the solution you have provided worked, thanks!
